webapp/tests/settings.py

```diff
@@ -57,7 +58,7 @@ def atexit_tmpremover(dirname):
     """ Utility to remove a temporary directory during program exit. """
     try:
         shutil.rmtree(dirname)
-        print("Removed temporary directory: %s" % dirname)
+        print(("Removed temporary directory: %s" % dirname))
```

Comment:
What is the second set of brackets for?

Reply:
Automatic tool being overenthusiastic. I'll fix it.